### DISCUSSION TITLE
Create-next-app creates a styles directory - What's the purpose of it?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#17109] [Help]
Hey, I just started using Next.JS and can't seem to figure out anywhere what the purpose of the `/styles` folder is. Anyone can explain it to me? I've seen that there are multiple ways of styling your app - but I can't seem to find anything specific to this folder using Google or the Documentation.

**Comment by M1ck0** [comment]
This is just a way to keep your files organized and separated. Pages go into `/pages`, components into `/components`, styles into `/styles` etc.

It is just a way to try to create some sort of convention that the whole community will use (which has its benefits). Of course, you don't have and don't need to follow this.

Hope this helps.

**Reply by ghost** [reply]
Do the same rules to use Sass, Less and styled component apply to this directory too? 
Does the old approach work too - having a folder that has a component with its style.
